preview/MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -137,6 +137,167 @@ int main(int argc, char * argv[])
             std::cout << numPackages << " Package(s) found" << std::endl;
             return S_OK;
         }
+        case OperationType::Unpack:
+        {
+            auto packageFilePath = cli.GetPackageFilePathToInstall();

```

Comment:
i'd recommend putting large chunks of code into separate functions/(possibly files). This can get unwieldy very fast

Reply:
probably 3 functions at least: UnpackPackage() UnpackBundle, and ApplyAcls(), and probaly best to put it in a separate file/class that handles Unpack

---
In reply to: [314106308](https://github.com/microsoft/msix-packaging/pull/201#discussion_r314106308) [](ancestors = 314106308)